README.md

```diff
@@ -52,7 +52,7 @@ You can run this repo virtually by using GitHub Codespaces or VS Code Remote Con
 1. Create a new folder and switch to it in the terminal
 1. Run `azd login` 
    1. If you're in multi-tenant organization, please use `azd login --tenant-id {YOUR_TENANT_ID}`
-   1. If you're using dev-container, please use `azd login --use-device-code`
+   1. If you're using VS Code DevContainer or GitHub Codespaces, please use `azd login --tenant-id {YOUR_TENANT_ID} --use-device-code`
```

Comment:
If you're using the VS Code DevContainer or GitHub Codespaces, please use `azd login --tenant-id {YOUR_TENANT_ID} --use-device-code`

Reply:
> If you're using the VS Code DevContainer or GitHub Codespaces, please use `azd login --tenant-id {YOUR_TENANT_ID} --use-device-code`

refined. #fd4cb76

Reply:
@jongio I'd done for the refinement. Please have time review. Thanks.

Reply:
Hi @jongio , any required needed from requester side?